mining.go

```diff
@@ -1900,6 +1898,12 @@ func (g *BlkTmplGenerator) UpdateBlockTime(header *wire.BlockHeader) error {
 	return nil
 }
 
+// UpdateBlockTime invokes `UpdateBlockTime` on the underlying
+// BgBlkTmplGenerator.
```

Comment:
BgBlkTmplGenerator -> BlkTmplGenerator.

Reply:
This method is exposed since rpcserver needs to access BlkTmplGenerator. Where to get a reference to the BlkTmplGenerator in that module without exposing it directly is not standing out at the moment, so I left it.

Reply:
I misunderstood this feedback when revisiting this PR. You were highlighting that the comment needs to change, not the method signature. Agreed that this should be altered, but given the timing and commits built on this already I think it might be best to handle in a later PR. Sorry I didn't catch and fix that earlier, and thanks for the review.

Reply:
I already updated it via #2276.